src/core/tensor/math_kernel.cu

```diff
@@ -184,6 +184,20 @@ __global__ void KernelAbs(const size_t n, const float *in, float *out) {
   }
 }
 
+__global__ void KernelAsType2(const size_t n, const float *in, int *out) {
```

Comment:
can you give a better name?
like KernelCastInt2Float or KernelInt2Float

Reply:
Thank you, this is rectified.